app/Console/Commands/SeederBackupData.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Storage;
+
+class SeederBackupData extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'db:seed:backup
+                            {--a|all : Whether to run through all available backups/restores in config}?
+                            {--remove : Remove backed up files}?
+                            {--restore : Restore the filament table}?
+                            {--t|table=* : Create/remove specific table file}?
+                            {--truncate : Truncate the table before restoring}';
```

Comment:
it looks like we're missing the entry for `--env`.

Reply:
`--env` is inherited by all commands already.